REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.3
 Polynomials
 Compat 0.4.1
+ProgressMeter
```

Comment:
Yes, right you all are. I was mostly experimenting here and forgot to start a new branch. I'm going to revert these changes and open a different PR elsewhere.